compiler/src/dotty/tools/dotc/typer/Typer.scala

```diff
@@ -2164,6 +2164,8 @@ class Typer extends Namer
       PrepareInlineable.registerInlineInfo(sym, rhsToInline)
 
     if sym.isConstructor then
+      if sym.is(Inline) then
+          report.error("constuctors cannot be `inline`", ddef)
```

Comment:
```suggestion
        report.error("constuctors cannot be `inline`", ddef)
```

Reply:
/cc @prolativ
Read "`report.error("constructors...`"
instead of "`report.error("constuctors...`".

Reply:
My bad